pyccel/decorators.py

```diff
@@ -98,3 +100,7 @@ def allow_negative_index(f,*args):
     def identity(f):
         return f
     return identity
+
+def kernel(f):
+    return numpy.array([[f]])
```

Comment:
I think we should add a comment explaining that this is written like this to allow the function to be indexed and the python code to execute without errors